Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM python:2.7
+RUN apt update
+RUN apt upgrade -y
+RUN apt install vim -y
```

Comment:
Can you combine this? 
`
RUN apt update && apt upgrade -y && apt clean
`

And why vim?